pom.xml

```diff
@@ -429,6 +430,13 @@
 		</dependency>
 	</dependencies>
 
+	<repositories>
+		<repository>
+			<id>jaspersoft-third-party</id>
```

Comment:
It is almost self-documented but I think it would be still good to add a comment here that lists the names of the dependencies requiring this repository.

Reply:
Makes a lot of sense. I added one.